### ISSUE TITLE
Stances/Forms as a condition when making rules

### ISSUE THREAD

**Issue by Mace2024** [xod-wow/LiteMount#223]
I want to be able to  use specific mounts only when in Moonkin/Astral form or Shadow Form, and sometimes in one of the paladin and warrior stances.

Ideally I would create a condition for this, except there is no option to have Forms as a condition when making rules

Is there a way to add this? or can this functionality be added - it would be really helpful. Unless there is a code to use in the advance section, help would be appreciated.

**Comment by mbattersby** [comment]
You can use any of the advanced conditions in rules by choosing "Advanced" as the rule condition and typing in the condition into the box.

https://github.com/xod-wow/LiteMount/wiki/Advanced-Options#conditions

The things you mention sound like they are all buffs, so you probably want to use the aura condition. For example, `aura:Crusader Aura`.

Happy adventuring, X.

**Comment by mbattersby** [comment]
I'm not sure of another way to say it. When you are making a rule, choose "Advanced" from the drop-down menu for the condition, and then to the right a text box will appear for you to type the advanced condition into.

**Comment by Mace2024** [comment]
> I'm not sure of another way to say it. When you are making a rule, choose "Advanced" from the drop-down menu for the condition, and then to the right a text box will appear for you to type the advanced condition into.

Funny, I tried that just after I responded yesterday, but  couldn't test it, - works so far. Didn't think it would be that simple. Awesome addon dude... thank you.  Aura was what I was missing as I had tried spell before and it didn't work.

On a side note, I think having forms/stance  as a condition option in the drop down menu would be great, I was going to ask if I could customise the add-on's code to add that  for my version.. what you think?
